tests/packages/mlx/test_mellanox-modules-compat.py

```diff
@@ -0,0 +1,19 @@
+# Requirements:
+# From --hosts parameter:
+# - host(A1): any master host of a pool, with access to XCP-ng RPM repositories and reports.xcp-ng.org.
+import logging
+from pkgfixtures import host_with_saved_yum_state
+
+def test_install_mlx_modules_alt(host_with_saved_yum_state):
+    host = host_with_saved_yum_state
+    logging.info(f"Installing mlx4-modules-alt alone")
+    host.yum_install(['mlx4-modules-alt'])
+    host.ssh(['modprobe', 'mlx4_ib'])
```

Comment:
It's `mlx4_en`. When in doubt due to incomplete specs, just ask :)

We're not sure `mlx4_ib` is of any use but we left it in the package as a build product.

```suggestion
    host.ssh(['modprobe', 'mlx4_ib'])
```